samples/sample_encode/src/sample_encode.cpp

```diff
@@ -335,7 +337,10 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
         else if (0 == msdk_strcmp(strInput[i], MSDK_STRING("-lad")))
         {
             VAL_CHECK(i+1 >= nArgNum, i, strInput[i]);
-            pParams->nRateControlMethod = MFX_RATECONTROL_LA;
+            if (!pParams->nRateControlMethod)
```

Comment:
If nRateControlMethod is already set, then it is not changed to MFX_RATECONTROL_LA. But nLADepth is set.
Thus parameter is ignored and no error is reported.
Now every new RC method in command line replaces previous. Not good to change it.